### PR TITLE
Document bolt metrics

### PR DESCRIPTION
**Pull request by raskchanky**
Note that by default, this does not render nicely, due to the very long metric names. Here's a screenshot of how it looks locally.

CleanShot%202021-07-15%20at%2015 15 25

Because of that, I added explicit line breaks to some of the metric names, just in an effort to keep the table from overflowing the width of the container. If this isn't the right way to solve this problem, let me know and I'm happy to do it differently. With the line breaks added, it looks like this:

CleanShot%202021-07-15%20at%2015 21 39

### CODE REVIEW

**Comment by raskchanky** [comment]
@jescalan I tagged you on this not for the content review but for the format review 😁

File: website/content/docs/internals/telemetry.mdx [hunk 1]
```diff
@@ -377,54 +377,72 @@ These metrics relate to the supported [storage backends][storage-backends].
...
+| `vault.raft-storage.transaction`                                       | Time to insert operations into a single log.                                                                                                                                                                      | ms                                | timer   |
+| `vault.raft-storage.entry_size`                                        | The total size of a Raft entry during log application in bytes.                                                                                                                                                   | bytes                             | sample  |
+| `vault.raft_storage.bolt.freelist.free_pages`                          | Number of free pages in the freelist.                                                                                                                                                   | bytes                             | sample  |
```
Comment (ncabatoff) [inline website/content/docs/internals/telemetry.mdx:428]: Are these really all samples?  Most of them are actually gauges I think.

Comment (raskchanky) [reply website/content/docs/internals/telemetry.mdx:428]: Whoops, copy/paste error on my part 🤦 . I was so focused on the formatting, I missed that. I'll get it fixed.

Comment (raskchanky) [reply website/content/docs/internals/telemetry.mdx]: f6c4a68f4f83657b5660647c705f9fc5f9b2da9f

File: website/content/docs/internals/telemetry.mdx [hunk 2]
```diff
@@ -377,54 +377,72 @@ These metrics relate to the supported [storage backends][storage-backends].
...
+| `vault.raft_storage.bolt.node.dereferences`                            | Number of node dereferences.                                                                                                                                                                                      | dereferences                      | gauge  |
+| `vault.raft_storage.bolt.rebalance.count`                              | Number of node rebalances.                                                                                                                                                                                        | rebalances                        | gauge  |
+| `vault.raft_storage.bolt.rebalance.time`                               | Total time spent rebalancing.                                                                                                                                                                                     | ms                                | sample  |
```
Comment (ncabatoff) [inline website/content/docs/internals/telemetry.mdx:440]: I prefer the phrasing we use for time-based samples elsewhere on this page "time taken to" vs "total time".  "Total time" for me would imply an ever-growing counter.

Comment (raskchanky) [reply website/content/docs/internals/telemetry.mdx]: ecfc7d3c2de6c04f68d41aa7efca4dc30a9a5132

**Review by ncabatoff — APPROVED** [review]

**Review by jescalan — APPROVED** [review]

**Pull request merged**